doc/nrf/ug_multi_image.rst

```diff
@@ -3,160 +3,120 @@
+Using multiple images has the following advantages:
 
-The same prefix convention is used to disambiguate targets. This means
-that to run menuconfig, for example, you can simply invoke the
-``menuconfig`` target to configure the top-level image and
-``mcuboot_menuconfig`` to configure the MCUBoot child image.
+* You can run the linker multiple times and partition the final firmware into several regions.
+  This partitioning is often useful for bootloaders.
+* Since there is a symbol table for each image, the same symbol names can exist multiple times in the final firmware.
+  This is useful for bootloader images, because it makes it possible to reuse libraries that the application image uses in the bootloader image.
+  (? is this correct? Original input was "This again is useful for bootloader images, as they often need to have their own fixed copies of the same libraries that the application image is using." ?)
+
+In the |NCS|, multiple images are required in the following scenarios:
+
+nRF9160 SPU configuration
+   The nRF9160 SiP application MCU is divided into a secure and a non-secure domain.
+   The code in the secure domain can configure the System Protection Unit (SPU) to allow non-secure access to the CPU resources that are required by the application, and then jump to the code in the non-secure domain.
+   Therefore, the nRF9160 samples (the parent image) require the :ref:`secure_partition_manager` (the child image) to be programmed in addition to the actual application.
+
+   See :ref:`zephyr:nrf9160_pca10090` and :ref:`ug_nrf9160` for more information.
 
-.. _application_defining_child_images:
+MCUboot bootloader
+   The MCUboot bootloader establishes a root of trust by verifying the next step in the boot sequence.
+   This first-stage bootloader is immutable, which means it must never be updated or deleted.
+   However, it allows to update the application, and therefore MCUboot and the application must be located in different images.
+   In this scenario, the application is the parent image and MCUboot is the child image.
 
-Defining new child images
-=========================
+   See :ref:`about_mcuboot` for more information.
+   The MCUboot bootloader is used in the :ref:`http_application_update_sample` sample.
 
-This section describes how to take an existing parent image and turn
-it into a child image that can be enabled. A parent image typically
-consists of source code, Kconfig fragments, and build script code. The
-source code and Kconfig fragments can be reused as-is, but several
-changes to the build scripts are necessary.
+nRF5340 support (? should we include this, even though it isn't using multi-image builds yet - it still needs several images ?)
+   nRF5340 contains two separate processors: a network core and an application core.
+   When programming applications to the nRF5340 PDK, they must be divided into two images, one for each core.
+   See :ref:`ug_nrf5340` for more information.
```

Comment:
It's a bit premature, but I'm sure this will be valid before this is released, so I think it makes sense to include it.